client/src/components/searchResults/index.js

```diff
@@ -1,31 +1,94 @@
 import React from 'react'
-import { FlatButton } from 'material-ui'
-import DisplayTable from './displayTable'
+import { FlatButton, Table, TableBody } from 'material-ui'
+import PropTypes from 'prop-types'
+import './style.css'
+import { renderUser, renderOrganization, renderProgram } from './rows'
 
-/* dummies */
-import dummyList from './dummies'
+let displaySearchKey
 
-import './style.css'
+const displaySearchKeyMessage = searchKey => {
+  if (displaySearchKey !== '') {
+    return <p>Results for {`"${searchKey}"`}</p>
+  }
+  return <p>Enter a search term</p>
+}
 
-const dummySearchKey = '"searchkey"'
-
-const searchResults = () => (
-  <div>
-    <div className="search-header">
-      <h1 className="search-title">Search Results</h1>
-      <p>Results for {dummySearchKey}</p>
-    </div>
-    <div className="search-box">
-      <div className="search-box-list">
-        <DisplayTable users={dummyList} />
-      </div>
-      <div className="search-back-to-top">
-        <FlatButton hoverColor="#bed62f" href="#top" fullWidth>
-          Return to top of results
-        </FlatButton>
+class SearchResults extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.renderTableRows = this.renderTableRows.bind(this)
+  }
+
+  componentWillMount() {
+    const { retrievedSearchKey, updateSearchKey } = this.props
+    if (retrievedSearchKey !== undefined) {
+      displaySearchKey = `${retrievedSearchKey}`
+    } else {
+      displaySearchKey = ''
+    }
+    updateSearchKey(displaySearchKey)
+  }
```

Comment:
This could be rewritten like this
```
if (retrievedSearchKey !== undefined) return updateSearchKey(retrievedSearchKey)

return updateSearchKey('')
```

Reply:
I just submitted some new commits that removed the searchKey from the reducer. It was causing too many issues for me.